webrender/src/device.rs

```diff
@@ -1374,19 +1364,19 @@ impl<B: hal::Backend> Program<B> {
         ]);
     }
 
-    pub fn bind_texture(&mut self, device: &B::Device, image: &ImageCore<B>, sampler: &B::Sampler, binding: &'static str) {
+    fn bind_texture(&mut self, device: &B::Device, set: &B::DescriptorSet, image: &ImageCore<B>, sampler: &B::Sampler, binding: &'static str) {
```

Comment:
while it might be OK to do this *for now*, in the long run we need to upstream some notion of descriptor sets, so that their recycling is cleaner, maybe even shove the responsibility to the renderer.